.vscode/launch.json

```diff
@@ -2,35 +2,34 @@
     // Use IntelliSense to learn about possible attributes.
     // Hover to view descriptions of existing attributes.
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-
     "version": "0.2.0",
     "configurations": [
-
         {
             "name": "Kitsune",
             "type": "python",
-            "request": "launch",
+            "request": "",
             "django": true,
             "program": "${workspaceFolder}/manage.py",
             "args": [
                 "runserver",
                 "0.0.0.0:8000"
             ]
         },
-
         {
             "name": "Kitsune (Gunicorn)",
             "type": "python",
-            "request": "launch",
+            "request": "",
             "django": true,
             "program": "/venv/bin/gunicorn",
             "args": [
                 "wsgi.app",
                 "--config",
                 "wsgi/config.py"
-            ]
+            ],
+            "autoreload": {
+                "enable": true
+            }
```

Comment:
This isn't a valid key from what I can see:

https://code.visualstudio.com/docs/python/debugging#_set-configuration-options